carsharing/trip-aggregator-app/deployment.yaml

```diff
@@ -0,0 +1,66 @@
+---
+# Source: streams-app/templates/deployment.yaml
```

Comment:
Should this file even be checked in? Looks to me like the result of `helm teplate`?

Reply:
no, you're right. Is deleted.